options.go

```diff
@@ -51,3 +51,15 @@ func (o maxStoredPartErrorsOption) apply(p *Parser) {
 	max := int(o)
 	p.maxStoredPartErrors = &max
 }
+
+// SetCustomParseMediaType if provided, will be used to parse media type instead of default ParseMediaType
```

Comment:
```suggestion
// SetCustomParseMediaType if provided, will be used to parse media type instead of the default ParseMediaType
// function.  This may be used to parse media type parameters that would otherwise be considered malformed.
```